tests/security/lynis/lynis_analyze_system_audit.pm

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Integrate the Lynis scanner into OpenQA: analyze the "system audit"
+#          current outputs with baseline
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#78224: poo#78230, poo#78330
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use lynis::lynistest;
+
+sub run {
+    my $dir           = $lynis::lynistest::testdir;
+    my $baseline_file = $lynis::lynistest::lynis_baseline_file;
+    my $current_file  = $lynis::lynistest::lynis_audit_system_current_file;
+    my $f_position_b  = $lynis::lynistest::f_position_b;
+    my $f_position_c  = $lynis::lynistest::f_position_c;
+
+    # Section name list
+    my @section_list_current  = ();
+    my @section_list_baseline = ();
+
+    select_console "root-console";
```

Comment:
Use serial console may have better performance, I'd suggest use serial if we don't have to use root-console

Reply:
> Use serial console may have better performance, I'd suggest use serial if we don't have to use root-console

To be honest I do not like "serial" console (tedious complains when doing openQA review and debuging). 
Btw, I used lots of perl the performance is quite good the run is very fast.

Reply:
Then no question here.